Sources/SwiftDocC/Infrastructure/Link Resolution/DocumentationCacheBasedLinkResolver.swift

```diff
@@ -78,8 +78,17 @@ final class DocumentationCacheBasedLinkResolver {
     func referenceFor(absoluteSymbolPath path: String, parent: ResolvedTopicReference) -> ResolvedTopicReference? {
         // Check if `destination` is a known absolute reference URL.
         if let match = referencesIndex[path] { return match }
-        
-        // Check if `destination` is a known absolute symbol path.
+
+        // Check if `destination` is a known absolute symbol path...
+        if !path.hasPrefix("/") && parent.pathComponents.count > 2 {
```

Comment:
I wasn't sure about this either. The documentation for that function lists the following resolution strategies:

```
     1. we check if the link is already resolved, i.e. the path is fully formed and there is a matching documentation node
     2. we check if the link is resolvable in the local context (e.g. the parent), i.e. "MyClass" ~> "MyClass/myFunction()"
     3. we check if the link is resolvable as a sibling to the parent, i.e. "MyClass/myFunction()" ~> "MyClass/path"
     4. we check if the link is resolvable in the root context (e.g. the module) of its parent, i.e. we will try resolving 'MyClass' as 'MyKit/MyClass'
     5. we check if there is a registered external resolver for the link's bundle id and if so, use that resolver
```

Since the list does not include checking in a **different** module, I figured it wasn't applicable to absolute references anyway and I also couldn't really find a situation in the context of my changes where the function was actually used.